docs/ContribOperators.md

```diff
@@ -2156,6 +2105,57 @@ This version of the operator has been available since version 1 of the 'com.micr
 </dl>
 
 
+### <a name="com.microsoft.MultiHeadAttention"></a><a name="com.microsoft.multiheadattention">**com.microsoft.MultiHeadAttention**</a>
```

Comment:
Assuming this needs a merge with main as the diffs don't seem related to this PR

Reply:
will do after GPU CI created the doc